src/item.cpp

```diff
@@ -7541,7 +7538,7 @@ void item::process_temperature_rot( float insulation, const tripoint &pos,
     time_point time;
     item_internal::goes_bad_cache_set( goes_bad() );
     if( goes_bad() ) {
-        time = std::min( { last_rot_check, last_temp_check } );
+        time = std::min( last_rot_check, last_temp_check );
```

Comment:
Since you asked about this: The braces are only needed if taking the `min` of more than two values.  I guess maybe at some point in the past there was an additional value in here.  Removing them should be safe.